imod_coupler/drivers/dfm_metamod/dfm_metamod.py

```diff
@@ -335,6 +369,73 @@ def exchange_stage_1d_dfm2mf6(self) -> None:
             updated_river_stage,
         )
 
+    def exchange_stage_2d_dfm2msw(self) -> None:
```

Comment:
can we make a unittest for this function?

Reply:
created issue IMOD6-1051 for this